config

```diff
@@ -31,7 +31,11 @@ case $ngx_wasm_runtime_name in
         ngx_wasm_runtime_srcs="$ngx_addon_dir/src/wasm/wrt/ngx_wrt_v8.c"
         ngx_wasm_runtime_lib_name="wee8"
         ngx_wasm_runtime_pre_libs="-lv8bridge"
-        ngx_wasm_runtime_post_libs="-ldl -lm -lpthread -lstdc++ -latomic"
+        ngx_wasm_runtime_post_libs="-ldl -lm -lpthread -lstdc++"
+
+        if [ "$NGX_WASM_RUNTIME_V8_NEEDS_LIBATOMIC" = 1 ]; then
```

Comment:
Could we automatize this with `$NGX_PLATFORM` instead? Nginx's `auto/` scripts already rely on it: https://github.com/nginx/nginx/blob/master/auto/os/conf#L22